pidgin/app.py

```diff
@@ -48,8 +48,12 @@ def get_metadata_dict(object_id):
     """
     response = request_metadata(object_id) # query to peregrine
     metadata = flatten_dict(response)
-    metadata['citation'] = generate_citation(metadata)
-    return remove_unused_fields(metadata) # translate the response to a dictionary
+
+    citation_fields = ['creator', 'updated_datetime', 'title', 'publisher', 'object_id']
```

Comment:
just for convention I would move this up to a global constant in this module, or have a `pidgin/constants.py` with this `CITATION_FIELDS` (and caps either way)